docs/content/ingestion/batch-ingestion.md

```diff
@@ -6,342 +6,19 @@ layout: doc_page
-```
-
-- Follow the instructions under "[Configure Hadoop for data
-loads](../tutorials/cluster.html#configure-cluster-for-hadoop-data-loads)" using the XML files from
-`/etc/hadoop/conf` on your EMR master.
-
-### Secured Hadoop Cluster
-
-By default druid can use the exisiting TGT kerberos ticket available in local kerberos key cache.
-Although TGT ticket has a limited life cycle, 
-therefore you need to call `kinit` command periodically to ensure validity of TGT ticket.
-To avoid this extra external cron job script calling `kinit` periodically,
- you can provide the principal name and keytab location and druid will do the authentication transparently at startup and job launching time.   
-
-|Property|Possible Values|Description|Default|
-|--------|---------------|-----------|-------|
-|`druid.hadoop.security.kerberos.principal`|`druid@EXAMPLE.COM`| Principal user name |empty|
-|`druid.hadoop.security.kerberos.keytab`|`/etc/security/keytabs/druid.headlessUser.keytab`|Path to keytab file|empty|
-
-#### Loading from S3 with EMR
-
-- In the `jobProperties` field in the `tuningConfig` section of your Hadoop indexing task, add:
-
-```
-"jobProperties" : {
-   "fs.s3.awsAccessKeyId" : "YOUR_ACCESS_KEY",
-   "fs.s3.awsSecretAccessKey" : "YOUR_SECRET_KEY",
-   "fs.s3.impl" : "org.apache.hadoop.fs.s3native.NativeS3FileSystem",
-   "fs.s3n.awsAccessKeyId" : "YOUR_ACCESS_KEY",
-   "fs.s3n.awsSecretAccessKey" : "YOUR_SECRET_KEY",
-   "fs.s3n.impl" : "org.apache.hadoop.fs.s3native.NativeS3FileSystem",
-   "io.compression.codecs" : "org.apache.hadoop.io.compress.GzipCodec,org.apache.hadoop.io.compress.DefaultCodec,org.apache.hadoop.io.compress.BZip2Codec,org.apache.hadoop.io.compress.SnappyCodec"
-}
-```
-
-Note that this method uses Hadoop's built-in S3 filesystem rather than Amazon's EMRFS, and is not compatible
-with Amazon-specific features such as S3 encryption and consistent views. If you need to use these
-features, you will need to make the Amazon EMR Hadoop JARs available to Druid through one of the
-mechanisms described in the [Using other Hadoop distributions](#using-other-hadoop-distributions) section.
-
-## Using other Hadoop distributions
-
-Druid works out of the box with many Hadoop distributions.
-
-If you are having dependency conflicts between Druid and your version of Hadoop, you can try
-searching for a solution in the [Druid user groups](https://groups.google.com/forum/#!forum/druid-
-user), or reading the Druid [Different Hadoop Versions](../operations/other-hadoop.html) documentation.
+Hadoop can be used for batch ingestion. The Hadoop-based batch ingestion will be faster and more scalable than the native batch ingestion. See [here](../ingestion/hadoop.html) for more details.
 
 ## Command Line Hadoop Indexer
 
 If you don't want to use a full indexing service to use Hadoop to get data into Druid, you can also use the standalone command line Hadoop indexer. 
```

Comment:
I feel like this 'option' for the command line Hadoop indexer makes more sense as a note at the beginning of the Hadoop indexing page.

Reply:
Moved this to the beginning of the hadoop page